addons/mrp/models/mrp_production.py

```diff
@@ -461,7 +461,7 @@ def _compute_reservation_state(self):
             relevant_move_state = production.move_raw_ids._get_relevant_state_among_moves()
             # Compute reservation state according to its component's moves.
             if relevant_move_state == 'partially_available':
-                if production.bom_id.operation_ids and production.bom_id.ready_to_produce == 'asap':
+                if production.workorder_ids.operation_id and production.bom_id.ready_to_produce == 'asap':
```

Comment:
Maybe do the same trick here https://github.com/odoo/odoo/blob/311d87e40c85ef80f80535ff5441caefff70fdee/addons/mrp/models/mrp_production.py#L730